src/service/repo/anonymousMessageRepo.ts

```diff
@@ -25,6 +26,7 @@ interface AnonymousMessageRepoTypes {
   sendAnonymousMessage: (
     channelId: string,
     message: string,
+    attachments: any,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same issue here with the `attachments` parameter. Avoid using `any` and use a more specific type.
```diff
-     attachments: any,
+     attachments: AttachmentType[],
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->